app/views/components/_download-link.html.erb

```diff
@@ -0,0 +1,9 @@
+<%
+  download_link_text ||= "Download File"
+%>
+
+<div class="app-c-download-link">
```

Comment:
@andysellick @fofr If we make the link block level, the focus state looks like this:
<img width="670" alt="screen shot 2017-07-24 at 09 40 32" src="https://user-images.githubusercontent.com/29889908/28515094-41055dea-7054-11e7-929d-5a0bf662a675.png">

rather than this?
<img width="667" alt="screen shot 2017-07-24 at 09 40 45" src="https://user-images.githubusercontent.com/29889908/28515101-46a2bdc4-7054-11e7-920b-5249f3cf53c4.png">